espnet2/bin/st_inference.py

```diff
@@ -306,7 +306,7 @@ def inference(
     token_type: Optional[str],
     bpemodel: Optional[str],
     allow_variable_data_keys: bool,
-    enh_s2t_task: bool,
+    enh_s2t_task: bool = False,
```

Comment:
```suggestion
    enh_s2t_task: bool,
```
We do not need this change with #4323

Reply:
OK, but acutally it seems can not pass the integration offline ST test with "enh_s2t_task: bool", therefore I add "enh_s2t_task: bool = False".